modules/sbo-about-service-binding-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/understanding-service-binding-operator.adoc
+
+:_content-type: CONCEPT
+[id="sbo-about-service-binding-operator_{context}]
```

Comment:
@Srivaralakshmi This ID isn't closed.

Reply:
Thanks @maxwelldb, Fixed it! Waiting for the build. Hope it resolves fine.